library/ix-dev/community/listmonk/templates/_configuration.tpl

```diff
@@ -0,0 +1,41 @@
+{{- define "listmonk.configuration" -}}
+  {{- $fullname := (include "ix.v1.common.lib.chart.names.fullname" $) -}}
+
+  {{- $dbHost := (printf "%s-postgres" $fullname) -}}
```

Comment:
Out of curiosity, can the name change ?

Reply:
The fullname is based on the name a user gives to the app, so it will be unique across the cluster.
`postgres` part is not changing, hardcoded in the template that generates the postgres pod.